test/e2e_node/podresources_test.go

```diff
@@ -711,7 +711,7 @@ var _ = SIGDescribe("POD Resources", framework.WithSerial(), feature.PodResource
 		})
 	})
 
-	ginkgo.Context("without SRIOV devices in the system", func() {
```

Comment:
xref https://github.com/kubernetes/kubernetes/pull/123398

May not need this one actually.